### ISSUE TITLE
Was this available in the Marketplace?

### ISSUE THREAD

**Issue by mikeworks** [Davuskus/elevator-music-vs#7]
I don't recall how I found this fun little piece of joy. Is this no longer something I could install on my newer machines?

**Comment by david98hall** [comment]
Hi! It was available on the marketplace before but I removed that account so I have to reupload it. I want to re-release it soon. 🙂

**Comment by mikeworks** [comment]
Yes! Sorry. I really enjoy this little bit of silliness with a dash of helpfulness - especially on longer publishes. Can I get you some sort of thanks?

Michael Fouquette
CenterLink | MikeWorks.NET | SFGMC UT2 Section Coordinator

I really enjoy Outlook for Android<https://aka.ms/ghei36> (get it)

________________________________
